packages/eslint-config-skuba/package.json

```diff
@@ -27,16 +27,16 @@
   },
   "dependencies": {
     "@types/eslint": "^8.4.1",
-    "@typescript-eslint/eslint-plugin": "^7.2.0",
-    "@typescript-eslint/parser": "^7.2.0",
+    "@typescript-eslint/eslint-plugin": "^7.14.1",
+    "@typescript-eslint/parser": "^7.14.1",
     "eslint-config-seek": "^13.0.0",
     "eslint-plugin-jest": "^28.0.0",
     "eslint-plugin-tsdoc": "^0.3.0",
     "eslint-plugin-yml": "^1.5.0"
   },
   "devDependencies": {
-    "eslint": "^9.0.0",
-    "typescript": "~5.4.0"
+    "eslint": "^8.56.0",
```

Comment:
Resolving complaints of non compatible ranges

Reply:
what do we need to get it back to 9?

Reply:
Looks like skuba itself is on 8 at the moment, so probably not urgent to resolve within this PR

Reply:
Wait for the eslint plugins to release support for 9 so it's less angry looking